consultation_analyser/lit/csr/silverDashboard/ThemeAnalysis/theme-analysis.lit.csr.mjs

```diff
@@ -0,0 +1,270 @@
+                                    .text=${html`
+                                        <iai-icon
+                                            .name=${"download"}
+                                        ></iai-icon>
+                                        <span>Export</span>
+                                    `}
+                                    .handleClick=${() => console.log("export initiated")}
+                                ></iai-silver-button>
+                            `}
+                        ></iai-silver-title>
+
+                        <div class="filters-row">
+                            <div class="filters">
+                                <iai-silver-select-input
+                                    .inputId=${"sort-value"}
+                                    .name=${"sort-value"}
+                                    .label=${"Sort by:"}
+                                    .hideLabel=${false}
+                                    .value=${this.sortType}
+                                    .options=${[
+                                        { value: "frequency", text: "Frequency" },
+                                        { value: "alphabetical", text: "A-Z" },
+                                    ]}
+                                    .handleChange=${(e) => this.setSortType(e.target.value)}
+                                    .horizontal=${true}
+                                ></iai-silver-select-input>
+
+                                <iai-icon-button .handleClick=${
+                                    () => this.setSortDirection(this.sortDirection === "ascending"
+                                        ? "descending"
+                                        : "ascending"
+                                )}>
+                                    <iai-icon
+                                        slot="icon"
+                                        .name=${this.sortDirection === "ascending"
+                                            ? "arrow_downward"
+                                            : "arrow_upward"
+                                        }
+                                    ></iai-icon>
+                                </iai-icon-button>
+                            </div>
+
+                            <div class="filters">
+                                <iai-silver-title
+                                    class="demographics-title"
+                                    .text=${"Demographics"}
+                                    .level=${3}
+                                ></iai-silver-title>
+
+                                ${Object.keys(this.demoData).map(category => {
+                                    const getSlug = (string) => string.toLowerCase().replace(" ", "-");
+
+                                    return html`
+                                        <iai-silver-select-input
+                                            .inputId=${getSlug(category)}
```

Comment:
Can we have some kind of 'select option' placeholder for the `<select>` filter elements? Otherwise I think it looks like filters might have been applied before there are any

<img width="1151" alt="screenshot of select items" src="https://github.com/user-attachments/assets/08998538-274f-462a-85f5-b0aac1ecb2d5" />